notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb

```diff
@@ -0,0 +1,986 @@
+{
```

Comment:
<p>ModelBatcgPredictOp and ModelUploadOp need to be upgraded to v1.</p><p> </p><p>Ex.</p><p>from google_cloud_pipeline_components.v1.model import ModelUploadOp</p><p>from google_cloud_pipeline_components.v1.batch_predict_job import ModelBatchPredictOp&nbsp;</p><p> </p><p>Move the GCPC imports into the pipeline definition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>